src/views/partials/board-settings.handlebars

```diff
@@ -0,0 +1,57 @@
+
+<!-- Css for dropdown menu on w3schools -->
+<div class="dropdown">
+    <button class="dropbtn">Settings</button>
+    <div class="dropdown-content">
+      <a href="#" class="btn btn-link">Board Settings</a>
+      <a href="#" class="btn btn-link">Invite</a>
+    </div>
+</div> 
+
+
+<div id="boardSettingsModal" class="modal">
+    <!-- Exit button (needs css) -->
+    <label for="Close Board Settings" class="close-btn">X</label>
+    
+    <!-- Form for changing the board settings -->
+    <form action="/board" method="POST">
+        <div class="boardNameDiv">
+            <label for="Board Name">Name: </label>
```

Comment:
the for attribute NEEDS to have the same text as the input's id (CHANGE FOR ALL)